src/components/MfaForm.js

```diff
@@ -1,48 +1,55 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import UserMessage from './UserMessage'
+import { Button, Field } from '@cwds/components'
 
-const MfaForm = ({maskedEmail, code, onCodeChange, onValidate, disableVerify, onCancel, errorMsg}) => {
+const MfaForm = ({maskedEmail, code, onCodeChange, onValidate, disableVerify, onCancel, errorMsg, countDown}) => {
   return (
     <form>
       <div id="div-forgot-password-msg">
         <UserMessage errorMessage={errorMsg}/>
-        <h1>Account Verification</h1>
+        <h1>Account Verification Required</h1>
         <br/>
-        <p>For additional security, we need to verify your account.</p>
+        <p>For additional security, we need to verify your account on this device.</p>
         <br/>
-        <p>We have sent a verification code by email to {maskedEmail}</p>
+        <p>An email was sent to {maskedEmail} with a unique verification code. Please enter that code below.</p>
         <br/>
-        <p>Please enter it below to complete verification.</p>
+        <p className='countDown'><b>Expires in: {countDown.min}:{countDown.sec}</b></p>
```

Comment:
If you simplify countdown to just a number, then the logic here becomes:
const min = Math.floor(countDown / 60);
const sec = countDown % 60; 
```suggestion
        <p className='countDown'><b>Expires in: {min}:{sec}</b></p>
```